server/src/main/java/org/elasticsearch/action/termvectors/TransportShardMultiTermsVectorAction.java

```diff
@@ -36,7 +36,8 @@
 import org.elasticsearch.threadpool.ThreadPool;
 import org.elasticsearch.transport.TransportService;
 
-public class TransportShardMultiTermsVectorAction extends TransportSingleShardAction<MultiTermVectorsShardRequest, MultiTermVectorsShardResponse> {
+public class TransportShardMultiTermsVectorAction extends TransportSingleShardAction<MultiTermVectorsShardRequest,
+        MultiTermVectorsShardResponse> {
```

Comment:
I don't like splitting this up if I can avoid it because you lose a lot of context. I prefer something like:
```
public class TransportShardMultiTermsVectorAction extends
        TransportSingleShardAction<MultiTermVectorsShardRequest, MultiTermVectorsShardResponse> {
```

Or even
```
public class TransportShardMultiTermsVectorAction extends TransportSingleShardAction<
        MultiTermVectorsShardRequest,
        MultiTermVectorsShardResponse> {
```